packages/atomic/src/components/common/atomic-icon/atomic-icon.stories.tsx

```diff
@@ -73,5 +72,9 @@ const {defaultModuleExport, exportedStory} = defaultStory(
   }
 );
 
-export default {...defaultModuleExport, title: 'Atomic/Icon'};
+export default {
+  ...defaultModuleExport,
+  title: 'Atomic/Icon',
+  id: 'atomic-icon',
+};
```

Comment:
So, yes it may seems that we could refactor the first argument of `defaultStory`and the id exported, but no: it breaks storybook.
We need to export it 'straight up'